### PR TITLE
Added LICENCE.md

### PR DESCRIPTION
**Pull request by Gander7**
Added the information for the MIT licence in LICENCE.md

### CODE REVIEW

**Comment by plusjamal** [comment]
Testing top level comment.

File: ProjectPrayerWall/LICENCE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (plusjamal) [inline ProjectPrayerWall/LICENCE.md]: Test line level comment.  Zach, I really think the right margin should be 500 characters wide for this file...

Comment (Gander7) [inline ProjectPrayerWall/LICENCE.md]: nope

File: ProjectPrayerWall/LICENCE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Copyright (c) 2017 ProjectPrayerWall
```
Comment (Gander7) [inline ProjectPrayerWall/LICENCE.md]: Thoughts on ProjectPrayerWall as the "fullname"?

File: ProjectPrayerWall/LICENCE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
```
Comment (plusjamal) [inline ProjectPrayerWall/LICENCE.md]: There is a 'start a review' button here..  I'm going to click it now to see wha tit does....

File: ProjectPrayerWall/LICENCE.md (hunk 1 above)
Comment (plusjamal) [inline ProjectPrayerWall/LICENCE.md]: lol.

File: ProjectPrayerWall/LICENCE.md (hunk 2 above)
Comment (plusjamal) [inline ProjectPrayerWall/LICENCE.md]: I like that.

File: ProjectPrayerWall/LICENCE.md (hunk 3 above)
Comment (plusjamal) [inline ProjectPrayerWall/LICENCE.md]: hmm....OK, it seems like I can click 'start review', and all my comments will appear together in a continuous block...i'm going to try it again..

File: ProjectPrayerWall/LICENCE.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
```
Comment (plusjamal) [inline ProjectPrayerWall/LICENCE.md]: I'm going to 'start a review' again with 3 comments...This is the first one..

File: ProjectPrayerWall/LICENCE.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
```
Comment (plusjamal) [inline ProjectPrayerWall/LICENCE.md]: This is the second comment...

File: ProjectPrayerWall/LICENCE.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
```
Comment (plusjamal) [inline ProjectPrayerWall/LICENCE.md]: This is a third comment....

Comment (plusjamal) [inline ProjectPrayerWall/LICENCE.md]: I see...

Comment (Gander7) [inline ProjectPrayerWall/LICENCE.md]: I think we can close this? though I'm hopgin this shows up on github

**Pull request closed without merging**